cloudfoundry-client-lib/src/main/java/org/cloudfoundry/client/lib/rest/CloudControllerRestClientImpl.java

```diff
@@ -1846,6 +1841,19 @@ private CloudTask createTask(UUID applicationGuid, CloudTask task) {
         return v3Client.tasks()
                        .cancel(request);
     }
+    
+    private Mono<? extends AbstractServiceKeyResource> createServiceKeyResource(CloudService service, String name,
```

Comment:
Change the return type to `Mono<? extends Resource<ServiceKeyEntity>>`, since that's the pattern in the rest of the methods.